test/unittests/t_catalog_manager.cc

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
But what is supposed to come into this file?  Isn't everything supposed to be in this file now in `t_catalog_mgr.cc`?